ui-v2/app/templates/dc/services/index.hbs

```diff
@@ -10,7 +10,7 @@
     {{/block-slot}}
     {{#block-slot 'toolbar'}}
 {{#if (gt items.length 0) }}
-        {{catalog-filter searchable=searchable search=filters.s status=filters.status onchange=(action 'filter')}}
+    {{#phrase-editor placeholder=(if (eq terms.length 0) 'service:name tag:name status:critical search-term' '') items=terms searchable=searchable}}{{/phrase-editor}}
```

Comment:
Why is the conditional around the placeholder required? I would assume the browser managed that based on the content?

Reply:
🤔 I think it's because the 'terms' (the little pill shapes) aren't actually in the input field, so only show the placeholder when `terms.length == 0`, otherwise you get a placeholder even if you have say 2 terms. I'll double check this though, but the more I read this the more I think that makes sense.